js/components/Icons/ContactMailIcon.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Svg, { G, Path, Circle } from 'react-native-svg';
+
+export const ContactMailIcon = ({ color }) => (
```

Comment:
Since there is only one component being exported, you should do export default here.